data/nosql/mongo-init.js

```diff
@@ -239,10 +239,16 @@ db.reports.insert([
     {"email": "admin-account@example.com", "name": "Monthly Report", "year": 2021, "month": 11, "total_visits": 100}
 ])
 
-
-
 db.composite_pk_test.insert([
     {"id_a":1, "id_b":10, "description":"linked to customer 1", "customer_id":"1"},
     {"id_a":1, "id_b":11, "description":"linked to customer 2", "customer_id":"2"},
     {"id_a":2, "id_b":10, "description":"linked to customer 3", "customer_id":"3"}
-    ])
\ No newline at end of file
+    ])
+
+//values to support test by specific objectId search
+
+db.type_link_test.insert([
+    {"_id":ObjectId("000000000000000000000001"), "name":"v1", "key":1, "email":"test1@example.com"},
+    {"_id":ObjectId("000000000000000000000002"), "name":"v2", "key":2, "email":"test1@example.com"},
+    {"_id":ObjectId("000000000000000000000003"), "name":"v3", "key":3, "email":"test1@example.com"}
+])
```

Comment:
This was a merge commit that also added other fixes - I'd generally separate these, and do merge commits by themselves.